client/task-list/list.js

```diff
@@ -54,15 +54,33 @@ export class TaskList extends Component {
 		this.possiblyTrackCompletedTasks();
 	}
 
+	getUngroupedTasks() {
+		const { tasks: groupedTasks } = this.props;
+		return [].concat(
```

Comment:
You can just do:

```
return [
  ...Object.keys( groupedTasks ).map(
				( taskType ) => groupedTasks[ taskType ]
			)
];
```

Reply:
I'm not sure about that. The `groupedTasks` is an object with two lists inside, one contains the setup tasks and the other one the extended tasks. I need the `concat` in order to merge the two lists. Otherwise, the result would be an array with 2 arrays with the split task lists inside.

Reply:
@octaedro gotcha I missed the structure there!

then let me suggest another way to simplify :D (not stressed about the change but this can be simpler)

```
return [...Object.values(groupedTasks)].flat()
```

Reply:
Nice catch @samueljseay !
That was my first option but I realized later that the project should work with [Node >= 8](https://github.com/woocommerce/woocommerce-admin/blob/main/package.json#L223) and `Array.flat` only will work with [Node > 10.23.0](https://node.green/#ES2019-features-Array-prototype--flat--flatMap--Array-prototype-flat), so the tests with lower versions of Node would fail.

Reply:
@octaedro this code doesn't need to work with Node 8. Any tests for this would be run in a browser like environment. For the browser Array.flat is polyfilled by corejs